src/capi-multiple-paidfor/web/index.scss

```diff
@@ -10,6 +10,40 @@
     color: $neutral-1;
 }
 
+.advert__title {
+  @include mq($until: tablet) {
+    width: 70%;
+    min-height: $gs-baseline * 5;
+  }
+}
+
+.badge {
+  display: inline-block;
+  @include mq($until: tablet) {
```

Comment:
How wide should we allow the badges to become? If we use `max-width` on the badge, and a `margin-right` on the advert-title, we can achieve the desired effect

flex is headache inducing; the image needs to be 100% width, and the badge logo is injected into the DOM by JS... 🤔